recipes/libpq/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "9.5.3" %}
+
+package:
+    name: libpq
+    version: {{ version }}
+
+build:
+  number: 0
+  skip: True   # [win]
+  # features:
+  #   - vc9               # [win and py27]
+  #   - vc10              # [win and py34]
+  #   - vc14              # [win and py35]
+  always_include_files:
```

Comment:
Just curious is this a new things or old and I never saw it before b/c I am lazy and I did not read the docs?